### PR TITLE
Add transform to handle aliasing edge-cases

### PR DESCRIPTION
**Pull request by danielsoutar**
Transforms unsupported expressions of the form `import X as Y` to a valid `using X: X as Y` expression. Also adds appropriate test with exposed `D` symbol.

### CODE REVIEW

**Comment by patrick-kidger** [comment]
We definitely shouldn't rewrite an `import` statement into a `using` statement. These are different things.

I think the only thing `@from` should emit is the same expression it was called with, but prepended with the appropriate lookup path.

(I used to have a nice reference on the ~8 different kinds of import statement in Julia -- import vs using; colon vs no colon; as vs no as; etc -- but I cant find it now.)

**Comment by Roger-luo** [comment]
just a comment on `import` I think `import` is just bad, most people should just not use it in Julia, only except `import ModuleName`, but it's a bit strange to change user program from `import` to `using` still. I think just support `using` is probably more consistent with what Julia have now.

**Comment by danielsoutar** [comment]
Hi @Roger-luo , @patrick-kidger , I had another go and was able to figure out how to do it without messing around with `using`. It's just a generalisation of the first case, and this passes the tests. Let me know what you think.

File: test/runtests.jl [hunk 1]
```diff
@@ -106,6 +106,14 @@ module wrapper12
 	end
 end
 
+module wrapper13
+	using FromFile
+	visible = [:D]
+	invisible = [:foo, :bar, :baz, :quux, :A, :B, :C]
+
+	@from "basic.jl" import A as D
+end
```
Comment (Roger-luo) [inline test/runtests.jl:115]: we need to be compatible with old julia

```suggestion
if VERSION ≥ v"1.6"
module wrapper13
	using FromFile
	visible = [:D]
	invisible = [:foo, :bar, :baz, :quux, :A, :B, :C]

	@from "basic.jl" import A as D
end

end # VERSION ≥ v"1.6"
```

**Comment by Roger-luo** [comment]
I think we can merge this first, I'll fix the 1.5 test

**Comment by Roger-luo** [comment]
thanks btw @danielsoutar
